01/implement-highest-rated.checkpoint.md

```diff
@@ -0,0 +1,81 @@
+# Implement Highest Rated
```

Comment:
So do you propose implementing PSEs with one just a lesson and the code block a checkpoint?

I like it, because it can show explanations.

Reply:
Yes, I was thinking that the first 3 questions would be autograded for completion, and that having the code challenge on a second page would really encourage students to do the first 3 steps before starting to code. Also thinking the entire PSE can be graded on a 0-3 scale (red-yellow-green)